data/assets/scene/solarsystem/sun/transforms_heliosphere.asset

```diff
@@ -1,10 +1,11 @@
 local sunTransforms = asset.require("scene/solarsystem/sun/transforms")
 local coreKernels = asset.require("spice/core")
 
+local heeq180Kernel = asset.resource("kernels/HEEQ180.tf")
+local heeqKernel = asset.resource("kernels/HEEQ.tf")
+local heeKernel = asset.resource("kernels/HEE.tf")
 
 
-local kernel = asset.resource("kernels/HEEQ180.tf")
-
 local HEEQ180ReferenceFrame = {
   Identifier = "HEEQ180ReferenceFrame",
```

Comment:
```suggestion
  Identifier = "HeliocentricEarthEquatorial180",
```

Reply:
Can't mark it in GitHub but the GUI.Name should be "Heliocentric Earth Equatorial"